shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -963,6 +963,21 @@ object GlutenConfig {
       .checkValues(Set("local", "heap-over-local"))
       .createWithDefaultString("local")
 
+  val COLUMNAR_VELOX_SPILL_MAX_LEVEL =
+    buildConf("spark.gluten.sql.columnar.backend.velox.spillMaxLevel")
```

Comment:
~~spillMaxLevel~~ -> maxSpillLevel

Since in our cpp code

https://github.com/oap-project/gluten/blob/4bd1b04650d19844da3f56430918fa3c2c2a92b8/cpp/velox/compute/WholeStageResultIterator.cc#L51